test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/RouteValueProviderTests.cs

```diff
@@ -74,8 +74,7 @@ public void GetValueProvider_ReturnsValue_UsesSpecifiedCulture()
             });
             var provider = new RouteValueProvider(BindingSource.Query, values, new CultureInfo("de-CH"));
 
-            // de-CH culture is slightly different on Windows versus other platforms.
-            var expected = TestPlatformHelper.IsWindows ? "31.10.2018 07:37:38 -07:00" : "31.10.18 07:37:38 -07:00";
+            var expected = "31.10.2018 07:37:38 -07:00";
```

Comment:
cc @dougbu

Reply:
👍

Might be worth a comment for future mergers, alerting developers that the `master` and `release/2.2` branches are _intentionally_ different here.